corehq/apps/reports/models.py

```diff
@@ -760,6 +760,9 @@ def send(self):
             self.delete()
             return
 
+        if self.owner.is_deleted():
+            return
```

Comment:
confirmed that this is actually a deleted mobile worker.  since this is permanent, makes sense to remove the reports: https://github.com/dimagi/commcare-hq/pull/9622